### DISCUSSION TITLE
Frozen Runes availability

### DISCUSSION THREAD

**Discussion by Undeadhunter** [chromiecraft/chromiecraft#6292] [General]
Due to the lack of Naxx 60, Forzen Runes are unavailable to acquire. 

One proposal would be to add them to roboto.

**Comment by kerricsson** [comment]
I'm supporting this suggestion because multiple CC guilds have bears as 2nd tanks in main roster, so bear resistance itemization is needed for further TBC progress.

**Comment by Hihilist** [comment]
How bad is the guide for Druid off-tanks from here? https://wowwiki-archive.fandom.com/wiki/Hydross_the_Unstable/Resistance_Gear

**Reply by Bassbone11** [reply]
using the guide on this server is flawed. while overall pointing you in the right direction, resilience means nothing for bears as we are already crit-immune from talents and we have no realistic idea of how much resistance we actually need as the calculations have been entirely changed. 365 resist now only resists 50% of damage on average instead of being a passive 75% mitigation like it was pre-3.0. if you're looking for a hybrid resist setup like they describe, [this](https://eightyupgrades.com/set/56LrZWYy4UXNe8a9F65oYW) is more or less a 'realistic' version of what you want. only 7% hit but the adds should be level 72 so its fine. you may even be able to get away with less resistance than shown but especially early on, the more you can get the better.
